spec/source/locales-currencies-tz.html

```diff
@@ -0,0 +1,163 @@
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-canonicalizelanguagetag">
+      <h1>CanonicalizeLanguageTag (locale)</h1>
+
+      <p>
+        The CanonicalizeLanguageTag abstract operation returns the canonical and case-regularized form of the _locale_ argument (which must be a String value that is a structurally valid BCP 47 language tag as verified by the IsStructurallyValidLanguageTag abstract operation). It takes the steps specified in RFC 5646 section 4.5, or successor, to bring the language tag into canonical form, and to regularize the case of the subtags, but does not take the steps to bring a language tag into "extlang form" and to reorder variant subtags.
+      </p>
+
+      <p>
+        The specifications for extensions to BCP 47 language tags, such as RFC 6067, may include canonicalization rules for the extension subtag sequences they define that go beyond the canonicalization rules of RFC 5646 section 4.5. Implementations are allowed, but not required, to apply these additional rules.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-defaultlocale">
+      <h1>DefaultLocale ()</h1>
+
+      <p>
+        The DefaultLocale abstract operation returns a String value representing the structurally valid (<emu-xref href="#sec-isstructurallyvalidlanguagetag"></emu-xref>) and  canonicalized (<emu-xref href="#sec-canonicalizelanguagetag"></emu-xref>) BCP 47 language tag for the host environment’s current locale.
+      </p>
+    </emu-clause>
+
+  </emu-clause>
+
+  <emu-clause id="sec-currency-codes">
+    <h1>Currency Codes</h1>
+
+    <p>
+      The ECMAScript 2015 Internationalization API Specification identifies currencies using 3-letter currency codes as defined by ISO 4217. Their canonical form is upper case.
+    </p>
+
+    <p>
+      All well-formed 3-letter ISO 4217 currency codes are allowed. However, the set of combinations of currency code and language tag for which localized currency symbols are available is implementation dependent. Where a localized currency symbol is not available, the ISO 4217 currency code is used for formatting.
+    </p>
+
+    <emu-clause id="sec-iswellformedcurrencycode">
+      <h1>IsWellFormedCurrencyCode (currency)</h1>
+
+      <p>
+        The IsWellFormedCurrencyCode abstract operation verifies that the _currency_ argument (which must be a String value) represents a well-formed 3-letter ISO currency code. The following steps are taken:
+      </p>
+
+      <emu-alg aoid="iswellformedcurrencycode-alg">
+        1. Let _normalized_ be the result of mapping currency to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
+        2. If the string length of _normalized_ is not 3, then
+          1. return `false`.
+        3. If _normalized_ contains any character that is not in the range *"A"* to *"Z"* (`U+0041` to `U+005A`), then
+          1. return `false`.
```

Comment:
- Merge this back with step 3. 
- The word "false" should be bold, regular font, not code/monospaced font